basicMysql/register/change.php

```diff
@@ -0,0 +1,121 @@
+<?php
+	session_start();
+  if(isset($_SESSION['login_user'])){
+	header("location: ../../index.php");
+  }
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Forgot-User</title>
+	<style>
+		<?php include "../../style.css" ?>
+	</style>
+
+  <script src="../../validity.js?newversion"></script>
+  <script type="text/javascript">
+    reg_obj = new Validity();
+    function checkUname(){
+      var user_name = document.getElementById('name').value;
+      reg_obj.checkName(user_name, "invalid_name", "submitBtn", "red");
+    }
+    function currentPassword(){
+      var current_pwd = document.getElementById('cpassword').value;
+      reg_obj.checkPasswords(current_pwd, "invalid_password", "submitBtn");
+    }
+    function newPassword(){
+      var new_pwd = document.getElementById('npassword').value;
+      reg_obj.checkPasswords(new_pwd, "invalid_newpassword", "submitBtn");
+    }
+    function rePassword(){
```

Comment:
The function name rePassword is not very clear, we should use a readable function name.